trigger-actions-framework/main/default/classes/TriggerActionFlowBypassTest.cls

```diff
@@ -83,13 +83,14 @@ private class TriggerActionFlowBypassTest {
 			myException,
 			'We should have an exception thrown in this scenario'
 		);
-		System.assert(
-			myException instanceof TriggerActionFlowBypass.TriggerActionFlowBypassObjectException,
+		Assert.isInstanceOfType(
```

Comment:
nice use of the new `Assert` class! 😎

Reply:
I like this method but _hate_ that the signature uses `(actual, expected)` as the ordering! It's very counter-intuitive.